components/elm/src/main/surfrdUtilsMod.F90

```diff
@@ -123,34 +123,35 @@ subroutine convert_cft_to_pft( begg, endg, cftsize, wt_cft )
     !        a crop landunit, and put them on the vegetated landunit.
     ! !USES:
```

Comment:
I was grepping through the code to see where we use this routine convert_cft_to_pft, to see if the changes would effect FATES somehow.  But I don't see where this is called.  Is it actually called anywhere?

Reply:
It isn't called anywhere in the current code. It is a placeholder for future functionality.